source/includes/baremetal/devices.md

```diff
@@ -0,0 +1,690 @@
+| `location.facility` <br/>_string_                            | A facility code. For example NYC1.                                                                                                       |
+| `deviceDetail.processor` <br/>_string_                       |                                                                                                                                          |
+| `deviceDetail.memory` <br/>_string_                          |                                                                                                                                          |
+| `deviceDetail.operatingSystem` <br/>_string_                 |                                                                                                                                          |
+| `deviceDetail.bandwidth` <br/>_string_                       |                                                                                                                                          |
+| `deviceDetail.internalNetwork` <br/>_string_                 |                                                                                                                                          |
+| `deviceDetail.ddos` <br/>_string_                            |                                                                                                                                          |
+| `deviceDetail.raidSetUp` <br/>_string_                       |                                                                                                                                          |
+| `deviceDetail.nextRenew` <br/>_string_                       |                                                                                                                                          |
+| `deviceDetail.deviceIPDetail.primaryIp` <br/>_string_        |                                                                                                                                          |
+| `deviceDetail.deviceIPDetail.description` <br/>_string_      |                                                                                                                                          |
+| `deviceDetail.deviceIPDetail.gatewayIp` <br/>_string_        |                                                                                                                                          |
+| `deviceDetail.deviceIPDetail.subnetMask` <br/>_string_       |                                                                                                                                          |
+| `deviceDetail.deviceIPDetail.usableIps` <br/>_Array[string]_ |                                                                                                                                          |
```

Comment:
There is no description for these fields.

Reply:
We don't have descriptions for these fields from the vendor.